client/passwd.go

```
@@ -70,6 +78,8 @@ func (cl *Client) sendToKPasswd(msg kadmin.Request) (r kadmin.Reply, err error)
 			return
 		}
 	}
+
 	err = r.Unmarshal(rb)
+
```

Looks like a lot of errors are ignored.